test/fixtures/scheduled_items.yml

```diff
@@ -6,6 +6,6 @@ standard:
   image: MyString
   name: MyString
   content: MyText
-  admission: MyString
-  audience: MyString
+  admission: ['Free', 'Tickets Required', 'Free - Registration Required', 'Sold out', 'Cancelled']
+  audience: ['Students', 'Club Members', 'Members', 'Faculty', 'Staff', 'Faculty and Staff', 'Chapman', 'Campus and Community', 'Orange Community', 'Public']
```

Comment:
This doesn't look right to me. Shouldn't admission and audience be a string value -- i.e. one of the choices from each list?